src/server/game/Entities/GameObject/GameObject.h

```diff
@@ -789,6 +789,8 @@ class TC_GAME_API GameObject : public WorldObject, public GridObject<GameObject>
         void AddLootMode(uint16 lootMode) { m_LootMode |= lootMode; }
         void RemoveLootMode(uint16 lootMode) { m_LootMode &= ~lootMode; }
         void ResetLootMode() { m_LootMode = LOOT_MODE_DEFAULT; }
+        void SetLootGenerationTime() { m_lootGenerationTime = time(NULL); }
```

Comment:
Should, but in this case its rather irrelevant (duration that is mostly measured in minutes)